src/hotrod/impl/protocol/HeaderParams.cpp

```diff
@@ -82,6 +82,10 @@ uint8_t HeaderParams::toOpRespCode(uint8_t code) {
         return HotRodConstants::EXEC_RESPONSE;
     case HotRodConstants::QUERY_REQUEST:
         return HotRodConstants::QUERY_RESPONSE;
+    case HotRodConstants::AUTH_MECH_LIST_REQUEST:
+        return HotRodConstants::AUTH_MECH_LIST_RESPONSE;
+    case HotRodConstants::AUTH_REQUEST:
+        return HotRodConstants::AUTH_RESPONSE;
```

Comment:
Uh, if you mean to write a code like this:   resp = req2resp[req], it probably pass the linker. A map would be the correct data struct, but I'm not sure about this solution that treats requests code as a kind of index: i.e. the array will be half empty. Well, just details, maybe we can open a new issue for this.